src/OrchardCore.Themes/TheAgencyTheme/Recipes/agency.recipe.json

```diff
@@ -90,59 +90,53 @@
           "DisplayName": "Liquid Page",
           "Hidden": false,
           "Settings": {
-            "Creatable": "True",
-            "Draftable": "True",
-            "Versionable": "True",
-            "Listable": "True",
-            "Securable": "False",
-            "Stereotype": null
+            "ContentTypeSettings": {
+              "Creatable": true,
+              "Draftable": true,
+              "Versionable": true,
+              "Listable": true
+            }
           },
           "ContentTypePartDefinitionRecords": [
             {
               "PartName": "LiquidPage",
               "Name": "LiquidPage",
               "Settings": {
-                "Position": "2"
+                "ContentTypePartSettings": {
+                  "Position": "2"
+                }
               }
             },
             {
               "PartName": "AutoroutePart",
               "Name": "AutoroutePart",
               "Settings": {
-                "ContentIndexSettings": {
-                  "Included": false,
-                  "Stored": false,
-                  "Analyzed": false,
-                  "Sanitized": false,
-                  "Tokenized": false,
-                  "Template": null
+                "AutoroutePartSettings": {
```

Comment:
These are all default values, and are handled by `GetSettings<T>` if they are null. So not needed in the recipe.